src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -81,7 +81,7 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
           <ApplicationView
             onUpdate={onUpdate}
             onSave={onSave}
-            validationState={validationState}
+            validationState={validationState || { isValid: true }}
```

Comment:
Maybe a good idea to add a comment here to explain. From my understanding there is still a problem happening, we just don't want to get much involved with the solution for now.

Reply:
Yeah okay. Done now in 4e35cdf

```
// TO-DO: ensure validationState gets calculated BEFORE rendering this child, so we don't need this fallback.
```